js/app/dashboard/redux/selectors.js

```diff
@@ -50,13 +51,16 @@ export const selectTaskLists = taskListSelectors.selectAll
 
 export const selectBookedUsernames = taskListSelectors.selectIds
 
+
 export const belongsToGroup = task => Object.prototype.hasOwnProperty.call(task, 'group') && task.group
-export const belongsToTour = task => Object.prototype.hasOwnProperty.call(task, 'tour') && task.tour
+export const belongsToTour = task => state => selectTaskIdToTourIdMap(state).has(task['@id'])
```

Comment:
I think it makes code a bit easier to understand if we keep using `belongsToTour` as a utility function, instead of `taskIdToTourIdMap.has(task['@id']` And call a selector something like `selectTourWithTask`

Reply:
yes it is not very nice indeed :/

you mean `selectTaskWithTour` no? because it is the problem at hand here

Reply:
we have a task and want to get("select") a tour which contains this task, correct? Hence, `selectTourWithTask` or maybe `selectTourByTask' or `selectTourContainingTask` something like this..

Reply:
should be `selectTourByTask` then in my understanding (?)

but in this case `belongsToTour` should be deleted no? because it has no use

Reply:
could be `selectTourByTask`

Reply:
`belongsToTour` could still be used as a util function, for example: `belongsToTour(taskIdToTourIdMap, task)` instead of `taskIdToTourIdMap.has(task['@id'])`

Reply:
OK I will unify this, it not very clean at the moment in the code (kind of always different each time we check)

https://github.com/coopcycle/coopcycle-web/issues/4061